.github/workflows/runE2ETest.yml

```diff
@@ -102,41 +85,25 @@ jobs:
       - name: Set THROTTLE_FACTOR for Windows
         if: matrix.os == 'windows-latest'
         run: echo "THROTTLE_FACTOR=5" >> $GITHUB_ENV
-      - name: Run test setup script
-        if: inputs.automationRepo == 'salesforcedx-vscode-automation-tests'
-        uses: coactions/setup-xvfb@b6b4fcfb9f5a895edadc3bc76318fae0ac17c8b3
-        with:
-          run: |
-            npm run setup
-          working-directory: 'salesforcedx-vscode-automation-tests'
-        env:
-          VSCODE_VERSION: ${{ matrix.vscodeVersion }}
-          SFDX_AUTH_URL: ${{ secrets.SFDX_AUTH_URL_E2E }}
-          ORG_ID: ${{ secrets.ORG_ID_E2E }}
       - name: Run headless test
         uses: coactions/setup-xvfb@b6b4fcfb9f5a895edadc3bc76318fae0ac17c8b3
         with:
           run: |
             npm run automation-tests
-          working-directory: ${{ inputs.automationRepo }}
+          working-directory: salesforcedx-vscode
         env:
```

Comment:
@madhur310 I think we need to pass the VSIX_TO_INSTALL value here to get rid of the error you shared in slack

Reply:
same for DEV_HUB_ALIAS_NAME